### ISSUE TITLE
Poor swipe detection on Samsung Galaxy S2

### ISSUE THREAD

**Issue by lewisdiamond** [hammerjs/hammer.js#814]
In a simple page with a full page div and only hammer js code:

```javascript
    var div = document.querySelector('.touch');
    var hammer = new Hammer.Manager(div);
    var x = 0;
    hammer.add( new Hammer.Swipe({treshold: 10, velocity:0.25, direction: Hammer.DIRECTION_HORIZONTAL}));
    hammer.on('swiperight', function (ev) {x = x + 1; div.textContent = x + " Swipe right: " + ev.type; });
    hammer.on('swipeleft', function (ev) { x = x + 1; div.textContent = x + " Swipe left: " + ev.type;}); 
```

I get about 70% detection, even with very low velocity and treshold. Sometimes it doesn't detect swipe multiple times in a row.

Works fine on the other devices I tested on.

**Comment by runspired** [comment]
What version of Android / Android browser does this S2 run?

you also have a typo `treshold` should be `threshold`, but that shouldn't be affecting this code's effectiveness.  Myself, I've generally found swipe to be hard to trigger at most velocities.  The velocity is such that almost always a pan is registered instead.

**Comment by lewisdiamond** [comment]
Model: GT-I9100

Cyanogenmod: 11-20141115-SNAPSHOT-M12-I9100
Android 4.4.4
Kernel 3.0.64-CM-g4ca83f

**Comment by arschmitz** [comment]
On other projects i work on we don't support moded phones because there is no end to the variables in play making it often impossible to reproduces issues. The SII only officially supports up to android 4.1 definitely not 4.4

**Comment by runspired** [comment]
Closed in favor of #838
